nodebuilder/tests/sync_test.go

```diff
@@ -130,13 +153,20 @@ Steps:
 6. Check FN is synced to height 30
 */
 func TestSyncFullWithBridge(t *testing.T) {
-	sw := swamp.NewSwamp(t)
-
-	bridge := sw.NewBridgeNode()
-
 	ctx, cancel := context.WithTimeout(context.Background(), swamp.DefaultTestTimeout)
 	t.Cleanup(cancel)
 
+	const (
+		blocks = 20
+		bsize  = 16
+		btime  = time.Millisecond * 300
+	)
```

Comment:
nit: DRY, seems like it is worthwhile defining these in a common place if both tests are using them.

Reply:
Fixed